app/assets/json/translations/locale-en.json

```diff
@@ -928,8 +928,9 @@
         "ESTIMATE_ELIGIBILITY": "Estimate Eligibility",
         "ESTIMATE_ELIGIBILITY_DESCRIPTION": "Tell us your household size and income and we'll highlight listings that might be a fit.",
         "NEW_LISTING_EMAIL_ALERT": "Get emailed whenever a new listing is posted.",
-        "SEE_LISTINGS": "See listings",
+        "SEE_RENTAL_LISTINGS": "Rent",
+        "SEE_SALE_LISTINGS": "Buy",
         "SIGN_UP_TODAY": "Sign Up today",
-        "TITLE": "Browse Affordable Housing Opportunities"
+        "TITLE": "Apply for affordable housing"
```

Comment:
I'm not sure what the M.O. usually is for updating text that's been translated. In this case it's only translated in Tagalog. @callaghanc do you know what we usually do?

Reply:
After talking to @chrisdolendo, she mentioned that she can add the Tagalog translation herself since she knows the language.

We have no standard process for updating translations. Historically, Barry has dictated translation requirements... I imagine there's an opportunity here for the DS team to set a higher baseline around language accessibility.

Chris mentioned she can also be responsible for hiding the "Buy" button with an env variable.